TombEngine/Game/Lara/lara_basic.cpp

```diff
@@ -771,7 +772,7 @@ void lara_as_pose(ItemInfo* item, CollisionInfo* coll)
 			return;
 		}
 
-		if (TrInput & IN_WAKE)
+		if (ACTION_HELD_WAKE)
```

Comment:
I don't fully understand the meaning of this, can you elaborate on both IN_WAKE and ACTION_HELD_WAKE meaning here?

Reply:
I refer to a "wake" input as something that will get Lara out of the AFK pose. `IN_WAKE` was the old version for legacy input bit fields, `ACTION_HELD_WAKE` is the new one.

Reply:
It looks like a #define, but under the hood it seems to call input action checks, I wonder if we should convert it into a class method? `if (WakeInputHappened())` or something would look more appropriate in my opinion.

Reply:
I can do that, but probably as another loose function, as it wouldn't work as a class method. But later when inputs are tied to a particular player/controller I could do it more cleanly.

Reply:
Yes, anyway calling it like a function seems to be better idea.